src/main/java/core/mate/academy/service/MachineProducer.java

```diff
@@ -1,11 +1,12 @@
 package core.mate.academy.service;
 
+import core.mate.academy.model.Machine;
 import java.util.List;
 
-public interface MachineProducer {
+public interface MachineProducer<T> {
     /**
      * In your implementations - create 2-3 models, add them to the list and return
      * @return - the list of models
      */
-    List<Object> get();
+    List<? extends Machine> get();
```

Comment:
In addition to previous comment 
```suggestion
    List<T> get();
```
Now, when u implement the class, and parameterize it with <Bulldozer>, method get, also can be parametrized only with bulldozer.

Reply:
++